src/uu/shuf/src/shuf.rs

```diff
@@ -42,15 +42,18 @@ mod options {
     pub static RANDOM_SOURCE: &str = "random-source";
     pub static REPEAT: &str = "repeat";
     pub static ZERO_TERMINATED: &str = "zero-terminated";
-    pub static FILE: &str = "file";
+    pub static FILE_OR_ARGS: &str = "file-or-args";
 }
 
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches = uu_app().try_get_matches_from(args)?;
 
-    let mode = if let Some(args) = matches.get_many::<String>(options::ECHO) {
-        Mode::Echo(args.map(String::from).collect())
+    let mode = if matches.get_flag(options::ECHO) {
+        match matches.get_many::<String>(options::FILE_OR_ARGS) {
+            Some(args) => Mode::Echo(args.map(String::from).collect()),
+            None => Mode::Echo(vec![]),
+        }
```

Comment:
I think this is slightly nicer, not by much though.

```suggestion
        Mode::Echo(
            matches
                .get_many::<String>(options::FILE_OR_ARGS)
                .unwrap_or_default()
                .map(String::from)
                .collect(),
        )
```